### PR TITLE
API-1457 Download logs button

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**
New feature on the connection part: we added a button Download logs for the connection with event subs. enabled

### CODE REVIEW

File: content/md/connectivity/manage-event-subscription.md [hunk 1]
```diff
@@ -85,3 +89,23 @@ That means that you will receive an event if someone updates the German descript
...
+
+To do so, we added a button to allow you to download logs related to an event subscription. 
+
+![Download logs button](../img/connection-download-logs-button.png)
+
+In the `.txt` file uploaded on your computer when you click on the `Download logs` button, you have access to several log levels:
+- **Error** logs show you when something went wrong with the connected application. Either it didn't answer quickly enough, or it responded with an error code. 
+- **Warning** logs alert you when your PIM reached the maximum number of events we can send per hour.
+- **Notice** logs tell you why an event hasn't been sent to your connected application. It can be because the connection itself raised the event through the REST API or because the connection has no permission to see the product in question. 
+- **Info** logs inform you when a bunch of events has been sent to your connected application.
```
Comment (tseho) [inline content/md/connectivity/manage-event-subscription.md:105]: Do we really need to expand on this ?
It's kinda overwhelming and in my opinion, the messages present in the logs are self explanatory.
Moreover, if we add new logs to the level `warning` for example, the documentation will be outdated and could be confusing.

I also think that the `Want to know more about each type of logs?` is good enough.

**Review by tseho — APPROVED** [review]

File: content/md/connectivity/manage-event-subscription.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ To ensure the proper functioning of your Akeneo PIM, you can connect **up to 3 c
 
 Now, let's move on to the URL configuration, and soon you'll be able to receive events.
 
-# Configuration
+# Configure
 
-## Request URL Configuration & Verification
+## Configure and verify your request URL
```
Comment (camillebourjade) [inline content/md/connectivity/manage-event-subscription.md:39]:
```suggestion
## Configure and confirm your request URL
```

File: content/md/connectivity/manage-event-subscription.md [hunk 3]
```diff
@@ -85,3 +89,18 @@ That means that you will receive an event if someone updates the German descript
...
+
+# Debug
+
+Now, everything is set-up, you probably want to be sure your event subscription is working well. 
+
+To do so, we added a button to allow you to download logs related to an event subscription. 
+
+![Download logs button](../img/connection-download-logs-button.png)
+
+In the `.txt` file uploaded on your computer when you click on the `Download logs` button, you have access to several log levels that will help you to understand why you did or didn't receive some events.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-event-subscription.md:101]:
```suggestion
When you click on `Download logs`, you have access to a `.txt`file containing several log levels that will help you to understand why you did or didn't receive some events.
```

File: content/md/connectivity/manage-event-subscription.md [hunk 4]
```diff
@@ -85,3 +89,18 @@ That means that you will receive an event if someone updates the German descript
...
+
+::: info
+Want to know more about each type of logs? 
+Please, read our article Debug and fix issues related to the Events API on the api.akeneo.com website.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-event-subscription.md:105]:
```suggestion
Please, read our article [Debug and fix issues related to the Events API](https://api.akeneo.com/events-documentation/subscription.html) on the api.akeneo.com website. 
```

**Review by camillebourjade — COMMENTED** [review]
English Fix

**Pull request merged**